src/lib/shutdown_manager.js

```diff
@@ -95,7 +95,7 @@ export default class ShutdownManager extends EventEmitter {
   }
 
   scheduleTerminationPoll() {
-    return async () => {
+    return (async () => {
```

Comment:
Maybe I'm missing something, but I think this function could be written like so:

```js
async scheduleTerminationPoll() {
  if (this.terminationTimeout) clearTimeout(this.terminationTimeout);

  let terminated = await this.host.getTerminationTime();

  if (terminated) {
    this.config.capacity = 0;
    this.emit('nodeTermination', terminated);
  }

  this.terminationTimeout = setTimeout(
    this.scheduleTerminationPoll.bind(this), this.nodeTerminationPoll
  );
}
```